README.md

```diff
@@ -1 +1,73 @@
-# cryptos-statistics
\ No newline at end of file
+# Cryptos Statistics
+
+![](https://img.shields.io/badge/Microverse-blueviolet) [![Hits](https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fgithub.com%2Flfmnovaes%2Ftodo-list&count_bg=%2379C83D&title_bg=%23555555&icon=&icon_color=%23E7E7E7&title=hits&edge_flat=false)](https://hits.seeyoufarm.com)
+
+> Web application to show data and prices for top Crypto Currencies.
+
+# Live Demo
+you can check out website [here](https://elastic-mestorf-f7a890.netlify.app/)
+
+
+## Preview
+
+![image](preview.png)
+
+
+## Built With
+
+- React
+- Redux
+- API
+- Material Ui
+
+
+
+## Getting Started
+Before we start, make sure you fit the prerequisites, the source code is inside the folder SRC/
+
+### Prerequisites
+- npm installed
+- VSCode
+- React-based on components
+
+### Setup
+- git clone https://github.com/iammouaz/cryptos-statistics.git
+
+### Install
+- `npm install`
+- `npm install --also=dev` (to install the dev dependencies, like webpack)
+
+### Usage
+- `npm start` from VSCode
+
+### Run tests
+- `npm tests`
+
+### Deployment
+- `npm deploy`
+
+## Authors
+
+👤 **Moaz El Molkey**
+
+- GitHub: [@iammouaz](https://github.com/iammouaz)
+- Twitter: [@MoazMulki1](https://twitter.com/MoazMulki1)
+- LinkedIn: [Mouaz El Molkey](https://www.linkedin.com/in/mohammad-mouaz-molki-1368981bb/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+To everyone in Microverse.
```

Comment:
After following the [design guide](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)),

- [ ] Please remember to acknowledge [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).